docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 
 services:
   db:
-    image: paradedb/paradedb:latest
+    image: postgres:15
```

Comment:
Shouldn't this be set to paradedb for the hybrid-search pr to work properly

Reply:
oh yeah good catch